Shop-1/src/index.ts

```diff
@@ -0,0 +1,21 @@
+import express from "express";
+import cors from "cors";
```

Comment:
import mongoose, {ConnectOptions} from "mongoose";
import dotenv from "dotenv";

I see you installed these packages, but you need to import them at the top of your server file